### ISSUE TITLE
FR: Allow for comments in custom dictionary

### ISSUE THREAD

**Issue by chrisgrieser** [tadashi-aikawa/obsidian-various-complements-plugin#23]
a simple comment syntax, could be useful, either in Obsidian's comment format `%% comment %%` maybe? This would make it possible to organize stuff in the dictionary.

```
%%coding%%
recursion
algorithm

%%writing%%
constructive
formality
```

**Comment by tadashi-aikawa** [comment]
I'll consider it later.

Until then, it would be great to create multiple dictionary files for each use cases :)

**Comment by tadashi-aikawa** [comment]
@chrisgrieser 
I have released as [v3.1.0](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/3.1.0).

For details, please show [README](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin#:~:text=Surrounding%20with%20%25%25%20means%20a%20comment%20(ex%3A%20%25%25This%20is%20a%20comment%25%25)).

**Comment by chrisgrieser** [comment]
works perfectly! 👌
